src/actions/lay.go

```diff
@@ -18,12 +18,16 @@ func Lay(
 ) {
 	if len(bricksToExecute) == 0 {
 		err = exinfra.ErrBadArg{Reason: "Error: you should specify at least a brick for lay action"}
+
 		return 3, err
-	} else if len(bricksToExecute) > 1 && conf.Interactive {
```

Comment:
We shouldn't ask confirmation if there is only one brick to lay or remove

Reply:
Well that's the behavior I find a bit weird; it's not consistent.

As a user, if I provide the flag `-i` (interactive, default behavior) or if I do not provide the flag `-I` (non-interactive), I expect to be asked for a confirmation, even if there's only a single brick involved.